classes/patterns/class-deepdivetopic.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Deep Dive Topic pattern class.
+ *
+ * @package P4GBKS
+ */
+
+namespace P4GBKS\Patterns;
+
+use P4GBKS\Patterns\Templates\Covers;
+
+/**
+ * Class Deep Dive Topic.
+ *
+ * @package P4GBKS\Patterns
+ */
+class DeepDiveTopic extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/deep-dive-topic';
```

Comment:
I think we agreed a while ago to add the `-pattern-layout` suffix to their names 🤔

Reply:
Hooo good point